ui/src/layout/Menu.js

```diff
@@ -10,6 +10,12 @@ import SubMenu from './SubMenu'
 import inflection from 'inflection'
 import albumLists from '../album/albumLists'
 import { HelpDialog } from '../dialogs'
+import { makeStyles } from '@material-ui/core/styles'
+
+const useStyles = makeStyles((theme) => ({
+  fixedSidebarOpen: { position: 'fixed' },
+  fixedSidebarClose: { position: 'fixed', width: '48px' },
```

Comment:
Why 48px? What it is used for? Should it be in `em` units?

Reply:
![132135](https://user-images.githubusercontent.com/19896788/114582515-46b58880-9c9e-11eb-8d6c-c3a3cc935e00.PNG)
Adding position fixed while the side menu closed sets some part visible merging with the grid as could be seen in the screenshot.
I agree with the px part we should use 3em and instead of setting fixed width of 3em we can add a ma width of 3em so it could be more flexible